client/public/index.html

```diff
@@ -35,6 +35,7 @@
     <meta
       name="description"
       content="Answers from the Singapore Government"
+      data-react-helmet="true"
```

Comment:
there's an outstanding issue to move to react-helmet-async. maybe we can move to that first?

Reply:
Okay, issue https://github.com/opengovsg/askgovsg/issues/22 is addressed in https://github.com/opengovsg/askgovsg/pull/429/commits/115944fe2300f7f43b76cd04d3a2fd426520f1fe.